pkg/fqdn/name_manager.go

```diff
@@ -313,10 +311,9 @@ func (n *NameManager) CompleteBootstrap() {
 // corresponding set of IPs has changed.
```

Comment:
nit: comment is wrong (affectedSelectors)

Reply:
Thanks, fixed!